ccdproc/image_collection.py

```diff
@@ -211,6 +211,10 @@ def keywords(self, keywords=None):
 
     @keywords.deleter
     def keywords(self):
+        """
+        versionchanged:: 1.3
```

Comment:
Has to be in the property not in the deleter. Also indentation and the `..` are important!